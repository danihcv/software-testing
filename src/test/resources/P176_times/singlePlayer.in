1 3
daniel 10
