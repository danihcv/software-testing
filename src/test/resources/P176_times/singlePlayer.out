Time 1
daniel

Time 2

Time 3
